### PR TITLE
Updated docs to fix product name and link to docs

### PR DESCRIPTION
**Pull request by RomanSek**
Updated docs to fix product name and link

@ampproject/wg-analytics

### CODE REVIEW

**Comment by RomanSek** [comment]
Hi, @micajuine-ho can you check this PR. It includes only update in docs.

File: extensions/amp-analytics/analytics-vendors-list.md [hunk 1]
```diff
@@ -463,11 +463,11 @@ Adds support for Pinpoll. Configuration details can be found at [pinpoll.com](ht
 
 Type attribute value: `piStats`
 
-### PPAS analytics
+### Piwik PRO Analytics Suite
 
 Type attribute value: `ppasanalytics`
 
-Adds support for PPAS analytics. Configuration details can be found at [piwik.pro](https://developers.piwik.pro/).
+Adds support for Piwik PRO Analytics Suite. Configuration details can be found at [piwik.pro](https://developers.piwik.pro/integrations/AMP_integration.html).
```
Comment (micajuine-ho) [inline extensions/amp-analytics/analytics-vendors-list.md:470]: Is this page still under development? I see this when I click on it:
image

**Review by micajuine-ho — COMMENTED** [review]
FYI: the analytics wg is deprecated. Please tag @ampproject/wg-components in the future :)

File: extensions/amp-analytics/analytics-vendors-list.md (hunk 1 above)
Comment (RomanSek) [reply extensions/amp-analytics/analytics-vendors-list.md]: You're right. I fixed the link.

**Review by micajuine-ho — APPROVED** [review]

**Comment by micajuine-ho** [comment]
Please rebase and merge, I think your PR Is behind main.

**Pull request merged**